### PR TITLE
Create ui-c8y-1020-29-0-configuration-list-migrated-to-data-grid.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3765](https://cumulocity.atlassian.net/browse/DM-3765)
Original PR: [7225](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7225)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-29-0-configuration-list-migrated-to-data-grid.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Configuration list migrated to data-grid
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-29-0-configuration-list-migrated-to-data-grid.md]: The Claude AI suggested to use the following title:
```suggestion
title: Configuration list migrated to data grid
```

File: content/change-logs/device-management/ui-c8y-1020-29-0-configuration-list-migrated-to-data-grid.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3765
+version: 1020.29.0
+---
+Configuration list migrated to data-grid
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-29-0-configuration-list-migrated-to-data-grid.md]: The Claude AI suggested to use the following body:
```suggestion
As part of ongoing improvements to the user interface, the configuration list has been migrated to a data grid. The data grid provides a more modern and user-friendly way to view and manage configuration items compared to the previous list view. This change affects all users who interact with configuration items. The data grid offers enhanced features such as sorting, filtering, and pagination, making it easier to find and work with specific configuration items.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1020-29-0-configuration-list-migrated-to-data-grid.md (hunk 1 above)
Comment (bozhidardedov) [inline content/change-logs/device-management/ui-c8y-1020-29-0-configuration-list-migrated-to-data-grid.md]:
```suggestion
title: Configuration repository migrated to data grid
```

File: content/change-logs/device-management/ui-c8y-1020-29-0-configuration-list-migrated-to-data-grid.md (hunk 2 above)
Comment (bozhidardedov) [inline content/change-logs/device-management/ui-c8y-1020-29-0-configuration-list-migrated-to-data-grid.md]:
```suggestion
Configuration repository has been migrated to a data grid. The data grid provides a more modern and user-friendly way to view and manage configuration items compared to the previous list view. The data grid offers enhanced features such as sorting, filtering, and pagination, making it easier to find and work with specific configuration items.
```

File: content/change-logs/device-management/ui-c8y-1020-29-0-configuration-list-migrated-to-data-grid.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Configuration repository migrated to data grid
```
Comment (MWindel) [inline content/change-logs/device-management/ui-c8y-1020-29-0-configuration-list-migrated-to-data-grid.md]:
```suggestion
title: Configuration repository migrated to a data grid
```

File: content/change-logs/device-management/ui-c8y-1020-29-0-configuration-list-migrated-to-data-grid.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3765
+version: 1020.29.0
+---
+Configuration repository has been migrated to a data grid. The data grid provides a more modern and user-friendly way to view and manage configuration items compared to the previous list view. The data grid offers enhanced features such as sorting, filtering, and pagination, making it easier to find and work with specific configuration items.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-29-0-configuration-list-migrated-to-data-grid.md]:
```suggestion
The configuration repository has been migrated to a data grid. The data grid provides a more modern and user-friendly way to view and manage configuration items compared to the previous list view. The data grid offers enhanced features such as sorting, filtering, and pagination, making it easier to find and work with specific configuration items.
```

**Review by BeateRixen — APPROVED** [review]

**Review by aaronraab — APPROVED** [review]

**Review by dafkk — APPROVED** [review]

**Pull request merged**